libraries/apollo-compiler/src/test/kotlin/com/apollographql/apollo/compiler/CodegenTest.kt

```diff
@@ -304,13 +305,15 @@ class CodegenTest {
               "URL" to ScalarInfo("java.lang.String", ExpressionAdapterInitializer("com.example.UrlAdapter.INSTANCE")),
               "ID" to ScalarInfo("java.lang.Long"),
               "String" to ScalarInfo("java.lang.String", ExpressionAdapterInitializer("new com.example.MyStringAdapter()")),
+              "ListOfString" to ScalarInfo("java.util.List<java.lang.String>"),
           )
         } else {
           mapOf(
               "Date" to ScalarInfo("java.util.Date"),
               "URL" to ScalarInfo("kotlin.String", ExpressionAdapterInitializer("com.example.UrlAdapter")),
               "ID" to ScalarInfo("kotlin.Long"),
               "String" to ScalarInfo("kotlin.String", ExpressionAdapterInitializer("com.example.MyStringAdapter()")),
+              "ListOfString" to ScalarInfo("kotlin.collections.List<kotlin.String?>"),
```

Comment:
That is quite the mouthful. I guess `List<String?>` doesn't work here?

Reply:
Just tried to make sure, but no, `ClassName.bestGuess()` expects a fully qualified name. I'll see if I can add a few shortcuts for `String`, `List`, etc.